solutions/repeatedChars.js

```diff
@@ -0,0 +1,35 @@
+/*# Repeated Characters
+
+Write a function called `repeatedChars` that receives two strings and returns a new string with the characters that are repeated in both input strings:
+
+```javascript
+repeatedChars("german", "gabriela"); // "gear"
+```
+*/
+/* Pseudocodigo
+Entrada: dos cadenas de caracteres
+Salida: caracteres repetidos
+
+Convertir una cadena en un objejo convirtiendo cada elemento en un key
+crear un vector vacío
+para cada elemento de la cadena dos 
+    si el objeto tiene una clave igual al elemento de la cadena de caracteres
+    agregar ese caracter al vector vacio
+
+devolver el vector
+
+*/
+
+var repeatedChars = function(str1, str2){
```

Comment:
Super, acá revisando esta solución, es O(n), realmente es O(3n) porque está el split, el reduce y el for. Pero se va la constante y queda O(n). Sin embargo, en espacio también es O(n)